ports/rp2/mpconfigport.h

```diff
@@ -36,7 +36,6 @@
 
 // Board and hardware specific configuration
 #define MICROPY_HW_MCU_NAME                     "RP2040"
-#define MICROPY_HW_ENABLE_UART_REPL             (0) // useful if there is no USB
```

Comment:
I think this line should be left here, and those below in this file not changed, because:
- That keeps things simple here, easier to understand the configuration.
- If a user wants to enable/disable USB/UART REPL then they should already be aware that they need UART if there is no USB.
- A user may not want a REPL at all.
- Eventually these config options should be wrapped in `#ifndef`'s so a board can configure them.

Reply:
Moved back if #ifndef.